addons/killtracker/initSettings.inc.sqf

```diff
@@ -0,0 +1,10 @@
+[
+    QGVAR(trackAI),
+    "CHECKBOX",
+    [LSTRING(TrackAI_DisplayName), LSTRING(TrackAI_Description)],
+    LSTRING(Category),
+    true,
+    false,
```

Comment:
I'm pretty sure this needs to be a global setting

Reply:
Just noticed that it's calling target events, so you would be right.